examples/crate_universe/WORKSPACE.bazel

```diff
@@ -9,6 +9,10 @@ load("@rules_rust//rust:repositories.bzl", "rust_repositories")
 
 rust_repositories()
 
+load("@rules_rust//crate_universe:bootstrap.bzl", "crate_universe_bootstrap")
+
+crate_universe_bootstrap()
```

Comment:
I think this will cause a second build of the resolver if `RULES_RUST_CRATE_UNIVERSE_RESOLVER_URL_OVERRIDE` is set? Can we sniff for that env var in the repository rule to make it no-op, rather than building a binary we'll then not use?

Reply:
Bootstrap will now only build if the environment variable is set.